content/scalers/azure-storage-blob.md

```diff
@@ -0,0 +1,73 @@
++++
+fragment = "content"
+weight = 100
+title = "Azure Storage Blob"
+background = "light"
++++
+
+Scale applications based on Azure Storage Blobs.
+
```

Comment:
We also provide additional information such as when it's available and whom maintains it:
```suggestion

**Availability:** v1.1+ | **Maintainer:** Community

```

Can we list you as owner of this scaler? Or do you want to support it as Microsoft @jeffhollan?

Reply:
I'm fine with any decision (list me as owner or Microsoft). Anyway we are going to use this in couple of heavy blob processing projects soon and I'm always available to give the feedback and help to update/maintain this.

Reply:
Well in that case I'd be happy to keep you as a maintainer, but don't want to pressure you!

If you want, feel free to use `**Maintainer:** Community (@IsharaPannila)`